lib/rules/no-async-operation.js

```diff
@@ -26,10 +27,11 @@ module.exports = {
     },
 
     create(context) {
+        const sourceCode = getSourceCode(context);
         return {
             CallExpression(node) {
                 const { callee } = node;
-                const scope = context.getScope();
+                const scope = sourceCode.getScope ? sourceCode.getScope(node) : context.getScope();
```

Comment:
```suggestion
                const scope = getScope(context, node);
```
Since we've got the helper for it. 🤷